client/src/components/faqBlock/faqconfig.json

```diff
@@ -1,33 +1,31 @@
         "Will there be food at the in-person event?": "There will only be food served for company representatives and volunteers.",
         "Who is the intended audience for this event?": "This fair is meant for students interested in the tech field and companies hiring for tech roles like Software Developer, Data Scientist, ML Engineer and more - not restricted to Engineering or CS students or 'purely' tech companies."
     },
     "Companies/Orgs": {
-        "How do I register and by when?": "You can register by reaching out to Umer at external@compeclub.com. The deadline is September 30th but the early pricing deadline is July 20th.",
-        "What is the Cancellation Policy?": "Grace Period: full refund will be granted if written notification of cancellation is received via email within 48-hours of registration. Cancellations received after 5:00 PM (MT) on December 15th 2023 are ineligible for grace period. Cancellations made outside of the 48-hour grace period but before 5:00 PM (MDT) on December 16th, 2022 will be charged a $100 cancellation fee to cover administration and processing costs. Cancellations made after 5:00 PM (MT) on December 15th, 2023 will be charged the full fee.",
-        "How early should I show up to set up my booth for the in-person fair?": "Please plan to be there by 11:30 AM at the latest so that we can open the doors to students at 12 PM. You can also ship any marketing/promotional material in advance. Please let us know by September 30th if you need this.",
-        "For when are students looking for a position?": "Most students will be looking for a Summer 2024 position (given when this event takes place) starting in May or June, however some students may be seeking a Fall 2024 internship or new grad position starting around May or September.",
+        "How do I register and by when?": "You can register by reaching out to us at external@compeclub.com. The deadline is September 30th but the early pricing deadline is July 20th.",
+        "What is the Cancellation Policy?": "Grace Period: full refund will be granted if written notification of cancellation is received via email within 48-hours of registration. Cancellations received after 5:00 PM (MT) on September 30th 2024 are ineligible for grace period. Cancellations made outside of the 48-hour grace period but before 5:00 PM (MDT) on September 30th, 2024 will be charged a $100 cancellation fee to cover administration and processing costs. Cancellations made after 5:00 PM (MT) on September 30th, 2024 will be charged the full fee.",
+        "How early should I show up to set up my booth/workshop for the in-person fair?": "Please plan to be there 30min before your scheduled section at the latest. If you are attending only the career-fair portion this would be 2:30pm MDT. You can also ship any marketing/promotional material in advance. Please let us know by September 30th if you need this.",
```

Comment:
```suggestion
        "How early should I show up to set up my booth/workshop for the in-person fair?": "Please plan to be there 30 min before your scheduled section at the latest. If you are only attending the career-fair portion this would be 2:30pm MDT. You can also ship any marketing/promotional material in advance. Please let us know by September 30th if you need this.",
```

Reply:
rephrased